### ISSUE TITLE
Allow adjustment of API endpoint paths.

### ISSUE THREAD

**Issue by enzanki-ars** [Unleash/unleash-client-python#4]
**Is your feature request related to a problem? Please describe.**
Currently, this project can not be used to integrate with GitLab as the API paths are different.  For example: `https://gitlab.com/api/v4/feature_flags/unleash/7760707/features` is the API for accessing the features, where `https://gitlab.com/api/v4/feature_flags/unleash/7760707` is the base API address, but this project attempts to use `https://gitlab.com/api/v4/feature_flags/unleash/7760707/api/client/features` instead, which is invalid for GitLab.  

**Describe the solution you'd like**
I would like to be able to configure the API url endpoints to be `/features` instead of the default of `/api/client/features`.  and etc. for each endpoint.

**Describe alternatives you've considered**
Changing the existing API constants works as expected, but not a frendly way to handle it across multiple systems where the project need to be configured.  Maintaining a fork seems kind of pointless.

**Additional context**
https://github.com/Unleash/unleash-client-python/blob/master/UnleashClient/constants.py

**Comment by ivanklee86** [comment]
@enzanki-ars TIL that GitLab has feature flags.  I'll take a look, though I'd like to poke around and see if I can get access for development purposes.

I suspect the answer is the hard-coded `/api/` section of the URL.  Would you know by chance whether the real URL is `<base_url>/features` and not `<base_url>/client/features`?

**Comment by enzanki-ars** [comment]
Not sure now.  I was able to get it to work with `<base_url>/features` in this code base, but `<base_url>/client/features` also returns a 401 as opposed to a 404.

**Comment by enzanki-ars** [comment]
It seems that either works, but I would assume that `/client/features` and etc. is the prefered version.  One thing I noticed is that GitLab docs show code using the official go verison of the client, and that one uses `/client/features` as the path (https://github.com/Unleash/unleash-client-go/blob/master/metrics.go#L139).  Based on that, I think it may be worth changing it so that `/api` is not included in the constants and instead passed in as the base URL. 

For example, compare the docs for the Python version:
```python
from UnleashClient import UnleashClient
client = UnleashClient("https://unleash.herokuapp.com", "My Program")
client.initialize_client()
```

And the Go version:
```go
unleash.Initialize(
	unleash.WithListener(&unleash.DebugListener{}),
	unleash.WithAppName("my-application"),
	unleash.WithUrl("http://unleash.herokuapp.com/api/"),
)
```

I will submit a pull request to make this more consistent.

**Comment by ivanklee86** [comment]
Thanks for the PR!  I've released the fix in ver `2.0.0` of the package. =)
